dashboard/serializers.py

```diff
@@ -40,23 +39,8 @@ def serialize(cls, program_enrollment):
         )
         return {
             'id': program.id,
-            'enrollments': edx_user_data.raw_enrollments,
-            'certificates': edx_user_data.raw_certificates,
-            'current_grades': edx_user_data.raw_current_grades,
-            'grade_average': cls.calculate_final_grade_average(mmtrack),
-            'is_learner': cls.is_learner(user, program),
+            'enrollments': cls.serialize_enrollments(edx_user_data.enrollments.queryset),
+            'grade_average': mmtrack.calculate_final_grade_average(),
+            'is_learner': is_learner(user, program),
             'email_optin': user.profile.email_optin,
         }
-
-    @classmethod
-    def is_learner(cls, user, program):
```

Comment:
it was not preventing me from implementing what i needed. i looked at the method, realized there was a better place for it, checked its usages in the code, determined that the impact on the PR was extremely minimal, and decided to move the method. all of this happened in a matter of seconds. i understand that we want to limit the size of PRs and i took that into account